docs/sources/setup-grafana/configure-security/configure-security-hardening.md

```diff
@@ -0,0 +1,97 @@
+---
+aliases:
+    - /docs/grafana/latest/setup-grafana/configure-security/configure-security-hardening/
+description: Security hardening enables you to apply additional security which might stop certain vulnerabilities from being exploited by a malicious attacker.
+title: Configure security hardening
+---
+
+# Configure security hardening
+
+Security hardening enables you to apply additional security which might stop certain vulnerabilities from being exploited by a malicious attacker.
+
+## Cookies
+
+If Grafana uses HTTPS, it is possible to further secure the cookie that is used as authentication to access the Web UI.
+
+> **Note:** Grafana must use HTTPS in order for the below configurations to work properly.
+
+### Secure cookie attribute
+
+Set the _cookie_secure_ option to `true`. This will enforce clients to set and send the cookie over a (valid HTTPS) secure connection to help mitigate against some MITM-attacks.
+
+Example:
+
+```toml
```

Comment:
which toml file?

Reply:
I added a note in the beginning regarding which file to be modified.